contrib/malware_name_mapping/README.md

```diff
@@ -0,0 +1,24 @@
+Malware Name Mapping
+====================
+
+This script fetches the current mapping file from the [Malware Name Mapping repository](https://github.com/certtools/malware_name_mapping/raw/master/mapping.csv) and converts it to the syntax needed by the modify expert.
+It expects a comma-separated file and uses the first column as regular expression and the second one as family name. Other URLs can be given as parameter.
+The resulting configuration sets the field `classification.identifier` to the malware family name corresponding to the given `malware.name`.
+
+By default, the result is printed to stdout, a filename can be given as parameter.
+
+For more information about the mapping project, have a look at the [Malware Name Mapping repository](https://github.com/certtools/malware_name_mapping).
+
+See also the built-in help of the script by calling it with `-h`.
+
+Default rule
+------------
+
+A default rule can optionally be added the the end of the rules with the `-d` flag.
+With this rule, the field `classification.identifier` is set to `malware.name` if the first is empty and the latter is not.
+As this is the last rule, it is kind of "last resort" to fill the identifier field.
+
```

Comment:
proposal: let's also in the docu make an example (cron entry) how to use the .py script. Think: copy & paste

Reply:
Another proposal ( I know this is a bit out of scope for this particular PR): make a check if the generated modify.conf file is actually valid (syntax check).

I am just thinking about the user-process here.

Reply:
> Another proposal ( I know this is a bit out of scope for this particular PR): make a check if the generated modify.conf file is actually valid (syntax check).

It's definitely valid json as we use the json lib here, so I will point the user to the `intelmq check` command.

Reply:
I added a section about automation